src/resource.py

```diff
@@ -40,6 +40,10 @@ def get_resource_type(resource_div):
             return 'folder'
         elif group == ['activity', 'assign', 'modtype_assign', '']:
             return 'assignment'
+        elif group == ['activity', 'url', 'modtype_url', '']:
+            # TODO what to do with other types?
+            if 'pdf' in resource_div.find('img')['src']:
+                return 'url'
         return 'other (e.g. quiz, forum, ...)'
```

Comment:
Could you add another group for archives (example: 
Einführung in die Theoretische Informatik (IN0011): "Templates for Hausaufgabe 2")?
It would be something like:
`elif group == ['activity', 'resource', 'modtype_resource', '']: ...`
Here we could distinguish between archives and txt files based on image source as it is used in the course "Einführung in die Theoretische Informatik (IN0011)".
What do you think?